src/CarbonAware.DataSources/CarbonAware.DataSources.Json/src/Configuration/JsonDataSourceConfiguration.cs

```diff
@@ -6,23 +6,13 @@ namespace CarbonAware.DataSources.Json.Configuration;
 /// <summary>
 /// A configuration class for holding Json Data config values.
 /// </summary>
-public class JsonDataConfiguration
+public class JsonDataSourceConfiguration
 {
-    private const string BaseDir = "data-files";
+    private const string BaseDir = "data-sources/json";
```

Comment:
NIT: `Dir` vs `Directory` consistency

```suggestion
    private const string BaseDirectory = "data-sources/json";
```